<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person Info</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbPersonInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>631</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Person Infor- Editor</string>
    </property>
    <widget class="QLineEdit" name="tfName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfDob">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lDob">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;DOB: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lWeight">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Weight:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lHeight">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Heigth:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfHeight">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Name: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfWeight">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bAddNewPerson">
     <property name="geometry">
      <rect>
       <x>494</x>
       <y>120</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Person</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Person Info - Database</string>
    </property>
    <widget class="QPlainTextEdit" name="taPersonList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>591</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bShowPersonDatabase">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Person DataBase</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DOB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew_Person">
   <property name="text">
    <string>New Person</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
